tests/regression/lp-1644439/kernel-new-enough.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# Check if the kernel is at least 4.4.0-67
+if [ "$(uname -r | perl -ne 'print $1 if /^([0-9]+)\./')" -lt 4 ]; then
+	exit 1
+fi
+if [ "$(uname -r | perl -ne 'print $1 if /^[0-9]+\.([0-9]+)\./')" -lt 4 ]; then
+	exit 1
+fi
+if [ "$(uname -r | perl -ne 'print $1 if /^[0-9]+\.[0-9]+\.[0-9]+-([0-9]+)/')" -lt 67 ]; then
+	exit 1
+fi
```

Comment:
How about
```sh
if ! uname -r | perl -ne '/^(\d+)\.(\d+)\.(\d+)-(\d+)/ or exit 1; exit 1 if $1<4; exit 1 if $2<4; exit 1 if $3==0 && $4<67'; then
    exit 1
fi
```
(you could write that without the `if`, because of the `-e`, but it makes it explicit)